### PR TITLE
Next prev but from terminal

### PR DESCRIPTION
**Pull request by BC12232**
Replacement for the request I just closed. This time I'm correctly merging into development. 
This version has the foward and backward button working for Library.tsx.

### CODE REVIEW

File: src/pages/Library.tsx [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/* QUESTION: How do u know what chapter it's on? Is there a bookmark?
```
Comment (matthewfala) [inline src/pages/Library.tsx:1]: Answered on slack, so not sure if you still want an answer for this. We have ILibraryContext state stored in the library component.

File: src/pages/Library.tsx [hunk 2]
```diff
@@ -299,12 +342,80 @@ class Library extends React.Component<ILibraryProps, ILibraryState>
...
+                                    Section: ({all: {Part: 0, SectionNumber: 0}, saying: {Part: 0, SectionNumber: curNum}, statement: {Part: 0, SectionNumber: curNum} }),
+                                    BrowseMode: (DefaultConfig.browseMode)
+                                };
+                                this.setContext(newContext);
+                                   
+                            }}
+                        >
+                            <IonIcon icon = {ellipsisVerticalOutline}></IonIcon>
+                        </IonButton>
                         <IonButton fill={"clear"} className="next-button"
```
Comment (matthewfala) [inline src/pages/Library.tsx:382]: If this render function gets long and ugly, you can always extract some components -> the components folder. https://reactjs.org/docs/components-and-props.html#extracting-components

File: src/pages/Library.tsx [hunk 3]
```diff
@@ -46,18 +55,21 @@ class Library extends React.Component<ILibraryProps, ILibraryState>
...
     private cm : ContentManager;
     private ref : any;
+    private INCREASE : number; //number of chapters we need to skip ahead, based on the back button
 
     constructor(props: ILibraryProps) {
         super(props);
 
         this.cm = this.props.contentManager;
         this.ref = React.createRef();
+        this.INCREASE = 0;
```
Comment (matthewfala) [inline src/pages/Library.tsx:65]: can we get rid of this?

**Review by matthewfala — CHANGES_REQUESTED** [review]

**Comment by BC12232** [comment]
Ok, changes fixed.

**Review by matthewfala — APPROVED** [review]

**Pull request merged**